explore-analyze/discover/background-search.md

```diff
@@ -4,7 +4,7 @@ mapped_pages:
   - https://www.elastic.co/guide/en/kibana/current/search-sessions.html
 applies_to:
   stack: preview 9.2
```

Comment:
should this mention something about 9.3 ga? or will the overall features still be in beta in 9.3?

Reply:
no status change it's still in preview